assets/src/edit-story/components/panels/design/preset/presetPanel.js

```diff
@@ -133,7 +119,7 @@ function PresetPanel({
   return (
     <Panel
       name={`stylepreset-${presetType}`}
-      initialHeight={Math.min(initialHeight, window.innerHeight / 3)}
+      initialHeight={getPanelInitialHeight(isColor, presets)}
       resizeable={resizeable}
       canCollapse={canCollapse}
     >
```

Comment:
Are we should the color preset panel should be initially expanded as it is now even when empty? It's taking up a lot of space (as can be seen in the many updated Percy snapshots).

Reply:
The height assigned in tests is set to `200` in the local storage for testing purposes, as if the user would have resized the panel like this.
The actual first-time calculated height is based on how many presets are in the panel, it won't take any extra room. Also, the default for panels is collapsed, meaning that it's only open if the user has opened it previously.